### ISSUE TITLE
Calibre-Web fails to reinstall (due to Ansible 2.8.6 lineinfile bug?)

### ISSUE THREAD

**Issue by holta** [iiab/iiab#2010]
Repeatedly re-running `./iiab-install` or `./runrole calibre-web` does not get past this error:

> TASK [calibre-web : Allow ImageMagick to read PDFs (debuntu)] ******************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'bool' object has no attribute 'expand'
fatal: [127.0.0.1]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-tmp-1571362652.83-61877404451192/AnsiballZ_lineinfile.py\", line 114, in <module>\n    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-tmp-1571362652.83-61877404451192/AnsiballZ_lineinfile.py\", line 106, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/root/.ansible/tmp/ansible-tmp-1571362652.83-61877404451192/AnsiballZ_lineinfile.py\", line 49, in invoke_module\n    imp.load_module('__main__', mod, module, MOD_DESC)\n  File \"/tmp/ansible_lineinfile_payload_A2hnR4/__main__.py\", line 567, in <module>\n  File \"/tmp/ansible_lineinfile_payload_A2hnR4/__main__.py\", line 558, in main\n  File \"/tmp/ansible_lineinfile_payload_A2hnR4/__main__.py\", line 338, in present\nAttributeError: 'bool' object has no attribute 'expand'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}

Reverting Ansible [2.8.6](https://launchpad.net/~ansible/+archive/ubuntu/ansible-2.8) to [2.7.14](https://launchpad.net/~ansible/+archive/ubuntu/ansible-2.7) does however work, if you do:

```
sudo su -
apt remove ansible
rm /etc/apt/sources.list.d/iiab-ansible.list
cd /opt/iiab/iiab
scripts/ansible-2.7.x
./runrole calibre-web
```

FYI here is the Ansible code (lineinfile) that is failing:
https://github.com/iiab/iiab/blob/master/roles/calibre-web/tasks/main.yml#L8-L15

Context: these results are from Ubuntu 18.04.3 (with graphical desktop) on an Intel NUC.

**Comment by holta** [comment]
I don't know if this same error is now breaking all MEDIUM-sized and BIG-sized fresh installs of IIAB...
i.e. all those attempting to install Calibre-Web :/

**Comment by holta** [comment]
Likely this is the upstream bug:

https://github.com/ansible/ansible/issues/63684 "lineinfile crash on 2.8.6"

**Comment by holta** [comment]
Installing the exact version of Ansible that you need can sometimes be a struggle.

As such, here are more/basic instructions to help you navigate Ansible uninstall and/or install issues:

> "What is Ansible and what version should I use?"
> http://FAQ.IIAB.IO Item 11

**Comment by holta** [comment]
> Likely this is the upstream bug:
> 
> [ansible/ansible#63684](https://github.com/ansible/ansible/issues/63684) "lineinfile crash on 2.8.6"

But this one might also be to blame?

https://github.com/ansible/ansible/pull/63194 "lineinfile - fix bug with insertbefore/insertafter and firstmatch"

**Comment by holta** [comment]
> > Likely this is the upstream bug:
> > [ansible/ansible#63684](https://github.com/ansible/ansible/issues/63684) "lineinfile crash on 2.8.6"
> 
> But this one might also be to blame?
> 
> [ansible/ansible#63194](https://github.com/ansible/ansible/pull/63194) "lineinfile - fix bug with insertbefore/insertafter and firstmatch"

And/or possibly...

https://github.com/ansible/ansible/issues/63756 "bug in lineinfile when using it with backrefs"

**Comment by holta** [comment]
Solved by IIAB's transition to Ansible 2.9.0, per:

#1996, PRs #2006 #2019 #2030
